src/app/modules/main/chat_section/module.nim

```diff
@@ -237,6 +238,7 @@ proc addCategoryItem(self: Module, category: Category, memberRole: MemberRole, c
         position = -1, # Set position as -1, so that the Category Item is on top of its Channels
         category.id,
         category.position,
+        categoryOpened = not category.collapsed,
```

Comment:
This fixes the issue with the category not being collapsed from the state on app start. Instead of calling a model update when we add a chat, we just use what's in the Dto